src/HealthGPS.Console/configuration.cpp

```diff
@@ -154,25 +156,26 @@ Configuration load_configuration(CommandOptions &options) {
         config.app_name = PROJECT_NAME;
         config.app_version = PROJECT_VERSION;
 
+        // Configuration root path
+        config.root_path = options.config_file.parent_path();
+
         // input dataset file
         auto dataset_key = "dataset";
         if (version == 1) {
             dataset_key = "file";
         }
 
         config.file = opt["inputs"][dataset_key].get<FileInfo>();
-        fs::path full_path = config.file.name;
-        if (full_path.is_relative()) {
-            full_path = options.config_file.parent_path() / config.file.name;
-            if (fs::exists(full_path)) {
-                config.file.name = full_path.string();
-                fmt::print("Input dataset file..: {}\n", config.file.name);
-            }
+        file_path = config.file.name;
+        if (file_path.is_relative()) {
+            file_path = config.root_path / file_path;
+            config.file.name = file_path.string();
```

Comment:
This same structure to make the path absolute is used in several places. I would suggest to write a small function that does the job and that you can reuse.